tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -0,0 +1,268 @@
+import json
+import time
+
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.utils.strings import short_uid
+from tests.aws.services.events.helper_functions import sqs_collect_messages
+from tests.aws.services.events.test_events import EVENT_DETAIL, TEST_EVENT_PATTERN_NO_SOURCE
+
+SOURCE_PRIMARY = "source-primary"
+SOURCE_SECONDARY = "source-secondary"
+
+
+class TestEventCrossRegion:
+    @markers.aws.validated
+    def test_event_bus_to_event_bus_cross_region(self, aws_client_factory, cleanups, snapshot):
+        primary_region = "us-east-1"
+        secondary_region = "eu-central-1"
```

Comment:
we could rather use the `region_name` and `secondary_region_name` fixtures here.

Reply:
fixed in: https://github.com/localstack/localstack/pull/10968/commits/6cdc36b96c5c0a8c15687c38ea2898711f92407d